js/main.js

```diff
@@ -1,2 +1,26 @@
-// var $searchForm = document.querySelector('#search-form');
-// $searchForm.addEventListener('submit', handleSubmit);
+var $searchForm = document.querySelector('#search-form');
+$searchForm.addEventListener('submit', handleSubmit);
+
+function selectElement(selector) {
+  return document.querySelector(selector)
+}
```

Comment:
I'm not sure if this function is really necessary, `document.querySelector` is already pretty clear on its purpose, there's no need to obfuscate what is happening by hiding it in a function.